src/main/kotlin/net/lumalyte/lg/LumaGuilds.kt

```
@@ -984,6 +984,32 @@ class LumaGuilds : JavaPlugin() {
+                logColored("✓ Guild Strikes hooked into LiteBans (punishments tracked per guild)")
+
+                // One-shot backfill of pre-existing LiteBans punishments
+                // (async, idempotent — deduped by LiteBans entry id).
+                Bukkit.getScheduler().runTaskAsynchronously(this, Runnable {
+                    try {
+                        val result = get().get<net.lumalyte.lg.infrastructure.litebans.StrikeBackfillService>().run()
+                        logColored(
+                            "✓ Guild Strikes backfill: ${result.recorded} recorded, " +
+                                "${result.attributed} attributed, ${result.skippedUnattributable} skipped (unattributable)",
+                        )
+                    } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is too generic. Prefer catching specific exceptions to the case that is currently handled.

(detekt.exceptions.TooGenericExceptionCaught)